deepchem/models/torch_models/ferminet.py

```diff
@@ -139,7 +139,8 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
             (one_electron_vector, one_electron_distance.unsqueeze(-1)), dim=3)
         one_electron = torch.reshape(one_electron.permute(0, 2, 1, 3),
                                      (self.batch_size, self.total_electron, -1))
-        one_electron_vector_permuted = one_electron_vector.permute(0, 2, 1, 3)
+        one_electron_vector_permuted = one_electron_vector.permute(0, 2, 1,
+                                                                   3).float()
```

Comment:
This is the minor type fix as ferminet layers requires its inputs to be in float data type, but the the one_electron_vector_permuted is a double datatype (from torch.permute)